### DISCUSSION TITLE
Mortgage Calculator Branch from Development

### DISCUSSION THREAD

**Discussion by Souky-11** [Souky-11/TafeT2S2T1VIRGO3#2] [General]
I have created a new feature branch called Mortgage Calculator . 
Do you think C# is the best IDE to use to develop programme? 
In terms of the calculations - do you think I need to convert years into months and then add the months together so that the calculations can work correctly?
Do you think it's a good idea to put placeholders in the textbox?

**Comment by Souky-11** [comment]
I have completed Section C and are you happy for to merge my feature branch into the development branch first?

**Reply by altheatafe** [reply]
Thanks for the update on the Mortgage Calculator feature branch!
- yes, I agree using C# language and Visual Studio is the popular IDE for C# development
- yes, converting years into months is indeed a good approach for mortgage calculations as it can enhance accuracy
- yes, i love the idea of adding placeholders in the textbox. It makes the interface more user-friendly

Feel free to merge our branch into the development branch. I am working on mine at the moment and will let you know when I am ready to merge. 

Regarding the Currency Conversion Calculator:
- Do you think it is good idea to display only 5 decimal places in the conversion rate?
- What  background colour do you prefer, plain white, light blue, or another option?
- What do you think about making the front size of "Amount $", "From" and "To" smeller than the title "Currency Conversion Calculator"?

**Reply by Souky-11** [reply]
Hi Althea
Regarding your questions with the Conversion Calculator:
I think the standard is 2 decimal places and it may be easier for the user to read 2 decimal places.  
I am not fussed with the background colour and I'm sure your design will look great regardless of what colour you use. 
I think making the font size of Amount, From and To smaller is a great idea and will make the page look better :) 

I hope that helps :)

**Comment by Souky-11** [comment]
I have completed my feature branch and have merged my feature branch into the development branch.  I have resolved the conflict and will push all the changes back to this remote repository. Are you okay with this?

**Reply by Souky-11** [reply]
I have completed the push of my development branch into this repos
